### ISSUE TITLE
Highwayman is missing

### ISSUE THREAD

**Issue by CD4NV** [alexbatalov/fallout2-ce#89]
After the first visit to the Den, a Highwayman isn't at the parking lot Smitty's: [SLOT03 (The Den, Highwayman).zip](https://github.com/alexbatalov/fallout2-ce/files/9177364/SLOT03.The.Den.Highwayman.zip)
As far as I remember, the car should be located near the Smitty house.

**Comment by alexbatalov** [comment]
I cannot reproduce it on the desktop (if I go from Klamath to Den it appears immediately). In your save game it's fixed if I exit to world map and return back. The same happens if I go to East Side, then come back to West Side (car is in the usual place). Are you playing on Vita?

**Comment by CD4NV** [comment]
> I cannot reproduce it on the desktop (if I go from Klamath to Den it appears immediately). In your save game it's fixed if I exit to world map and return back. The same happens if I go to East Side, then come back to West Side (car is in the usual place). Are you playing on Vita?

Yeah, on PS Vita.

**Comment by DvaMishkiLapa** [comment]
I think it's time to introduce some query templates for productivity. There will be many of us. Vita has two ports, one of which is slightly more relevant. But still, the versions of the ports used may be different.

**Comment by Northfear** [comment]
I'll try setting up some auto-builds for PSV today, because the situation is getting messy indeed with the number of fixes and releases

**Comment by DvaMishkiLapa** [comment]
I own a PS Vita, and I have the Steam version of fallout. If you ever need any help. Although, there are more and more of us here :D

**Comment by Northfear** [comment]
@CD4NV @DvaMishkiLapa Vita development builds are here
https://github.com/Northfear/fallout2-ce-vita/releases/tag/latest
Some bugs were fixed since the last release, so it's better to double check them

**Comment by CD4NV** [comment]
> 

Thx a Lot.

**Comment by CD4NV** [comment]
> @CD4NV @DvaMishkiLapa Vita development builds are here https://github.com/Northfear/fallout2-ce-vita/releases/tag/latest Some bugs were fixed since the last release, so it's better to double check them

Tested this version: The problem with the dog (Smoke) and Sulik is solved, but other problems have arisen:
1. The game began to fall very often - almost a constant crash (C2-12828-1) when trying to move to a new location or city.
2. Stable crash when trying to talk to "talking heads" like Sulik.

I couldn't check other errors, for example with Highwayman, because the game immediately crashes when trying to log in on the Den.
If you need the save files, let me know so that I can prepare them and send them to you. Thanks.

**Comment by DvaMishkiLapa** [comment]
> @CD4NV @DvaMishkiLapaСборки для разработки Vita находятся здесь https://github.com/Northfear/fallout2-ce-vita/releases/tag/latest . Некоторые ошибки были исправлены с момента последнего релиза, поэтому лучше перепроверить их.

1. Highwayman stands still, waiting to be bought and repaired (https://github.com/alexbatalov/fallout2-ce/issues/89#issue-1316113079);
2. Trying to load a save from the menu during the game - `C2-12828-1`. At the same time, the save selection in the start menu works.;
3. https://github.com/alexbatalov/fallout2-ce/issues/90#issue-1316168176 is still there (I checked for interest);
4. Yes, talking heads leading to mistakes (e.g., Vic the merchant).

**Comment by Northfear** [comment]
`-fpeephole2` seems to be causing a lot of crashes on PSV in heap related code. I've tweaked CFLAGS a bit. Latest dev build seems to be much more stable

**Comment by CD4NV** [comment]
> `-fpeephole2` seems to be causing a lot of crashes on PSV in heap related code. I've tweaked CFLAGS a bit. Latest dev build seems to be much more stable

It has become much better. There are almost no departures. Almost all the early errors have been fixed: Smoke, Flint, Sulik - everything is fine, even the Highwayman is in the parking lot.
The only nuance that I noticed is that some old saves just don't load (apparently, this is due to the fact that these saves were made on previous builds).

I also noticed one curious moment:
When I tried to jump into the Temple of Trials (from Arroyo in), a menu in the style popped up: "Are you ready to take the test?" All my things were sent to the trunk and the door was locked. I couldn't crack it. I don't remember if this bug was in the original version, but it looks ridiculous when you are offered to take the test again =) Maybe this is due to the fact that when I passed the test in the Temple, I did not pass the final test honestly, I just stole the key and went out into the Arroyo.
